composer.json

```diff
@@ -67,7 +67,8 @@
             "AsyncAws\\Illuminate\\Mail\\": "src/Integration/Laravel/Mail/src",
             "AsyncAws\\Illuminate\\Queue\\": "src/Integration/Laravel/Queue/src",
             "AsyncAws\\SimpleS3\\": "src/Integration/Aws/SimpleS3/src",
-            "AsyncAws\\Symfony\\Bundle\\": "src/Integration/Symfony/Bundle/src"
+            "AsyncAws\\Symfony\\Bundle\\": "src/Integration/Symfony/Bundle/src",
+            "AsyncAws\\Rekognition\\": "src/Service/Rekognition/src"
```

Comment:
Put it just over S3 to keep this in alphabetical order and next to the other clients.